arrayfire_wrapper/_backend.py

```diff
@@ -191,22 +266,26 @@ def _lib_names(self, name: str, lib: _LibPrefixes, ver_major: str | None = None)
         post = self._backend_path_config.lib_postfix if ver_major is None else ver_major
         lib_name = self._backend_path_config.lib_prefix + lib.value + name + post
 
-        lib64_path = self._backend_path_config.af_path / "lib64"
-        search_path = lib64_path if lib64_path.is_dir() else self._backend_path_config.af_path / "lib"
-
-        site_path = Path(sys.prefix) / "lib64" if not is_arch_x86() else Path(sys.prefix) / "lib"
-
-        # prefer locally packaged arrayfire libraries if they exist
-        af_module = __import__(__name__)
-        local_path = Path(af_module.__path__[0]) if af_module.__path__ else Path("")
+        lib_paths = [Path("", lib_name)]
 
-        lib_paths = [Path("", lib_name), site_path / lib_name, local_path / lib_name]
+        # use local or site packaged arrayfire libraries if they exist
+        try:
+            local_path = _find_site_local_path()
+            lib_paths.append(local_path / lib_name)
+        except ValueError as e:
```

Comment:
Please, stop catching the error to optionally do nothing 💀 
Just follow the main flow and make sure the `_find_site_local_path` returns a string, e.g., an empty string, so you won't need to catch any errors.